src/plugins/workspace/server/utils.ts

```diff
@@ -50,3 +54,34 @@ export const getPrincipalsFromRequest = (
 
   throw new Error('UNEXPECTED_AUTHORIZATION_STATUS');
 };
+
+export const updateDashboardAdminStateForRequest = (
```

Comment:
@ruanyl and I think perhaps we should set `isDashboardAdmin` to true if groups.length === 0 && users.length === 0. They will be empty when permissionControl flag is open but authentication is not enabled, in that case, everyone should be dashboard admin.

Reply:
Sure, the logic has been updated. If security plugin uninstall and permission control enabled, login defaults to OSD admin.